Library/Formula/docker-machine.rb

```diff
@@ -24,6 +24,10 @@ def install
 
     system "make", "build"
     bin.install Dir["bin/*"]
+
+    bash_completion.install "contrib/completion/bash/docker-machine.bash"
+    bash_completion.install "contrib/completion/bash/docker-machine-prompt.bash"
+    bash_completion.install "contrib/completion/bash/docker-machine-wrapper.bash"
```

Comment:
Did you use `Dir[]`? It shouldn’t be `bash_install "contrib/completion/bash/*.bash"` but `bash_install Dir["contrib/completion/bash/*.bash"]`.